estimation/vision/fiducial_pose.hh

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <cassert>
+#include <vector>
+
+//%deps(opencv)
+#include <opencv2/opencv.hpp>
+#include <opencv2/aruco.hpp>
+
+#include "sophus.hh"
+
+namespace estimation {
+namespace vision {
+
+struct MarkerDetection {
+  SE3 marker_center_from_camera;
+  int id;
+};
+
+struct MarkerInWorld {
+  SE3 world_from_marker;
+  int id;
+};
+
+std::vector<MarkerDetection> detect_markers(cv::Mat mat);
+
+std::vector<MarkerInWorld> get_world_from_marker_centers(cv::Mat camera_image,
+                                                         SE3 world_from_opengl_camera);
```

Comment:
take `const SE3&`

Reply:
Also change `world_from_opengl_camera` here to `world_from_camera`

Will we actually be able to use this function on the vehicle (Since this requires knowing world_from_camera already)